### DISCUSSION TITLE
Synchronizing existing instance level objects without using -force to update object properties that are different

### DISCUSSION THREAD

**Discussion by simple2code** [dataplat/dbatools#9046] [Q&A] [answered]
I want to synchronize logins/jobs across multiple sql server instances. If an object is not in the destination, I use dbatools that works great to copy.  However, if the "object exists", I need a way to update the destination object if any of the properties are different from the source object. The -force option per docs will do a delete/re-create which means the job history is lost. Ideas?

**Comment by wsmelton** [comment, marked as answer]
We currently have no commands for syncing any given object. There are commands available for you to create wrappers to accomplish what you need for your environment (e.g., Get object, compare object, apply changes that don't match from source).

**Reply by simple2code** [reply]
thx :)
